### PR TITLE
Add link to Flowee Pay

### PR DESCRIPTION
**Pull request by zander**

### CODE REVIEW

**Comment by imaginaryusername** [comment]
Are standalone release binaries of some sort coming anytime soon? I suppose the .deb files buried inside the PPA are technically binaries, but people usually expect a bit more from a released wallet.

**Comment by zander** [comment]
There are releases for 2 of the most used Linux distros, in their native method of releases. "The unix way" is followed where we go to those linux distros instead of trying to create a walled-garden which people have to download from us, which is what Windows has as standard.

What is shipped is exactly what those using these linux distros expect and they would really not want to download an executable from a random website. **Especially not** for something that holds their money.

File: app/wallets.html [hunk 1]
```diff
@@ -45,6 +45,7 @@ <h2>$( wallets.desktop.title )$</h2>
         <a class="d-block" href="https://pokket.cash" target="_blank">Pokket</a>
         <a class="d-block" href="https://melis.io" target="_blank">Melis Wallet</a>
         <a class="d-block" href="https://paytaca.com" target="_blank">Paytaca (Chrome only)</a>
+        <a class="d-block" href="https://flowee.org/products/pay/" target="_blank">Flowee Pay</a>
```
Comment (imaginaryusername) [inline app/wallets.html:48]: Recommend adding (Linux only) after the name

Comment (imaginaryusername) [reply app/wallets.html]: https://github.com/bchinfo/bch.info/pull/161/commits/50fea161fb36c21137c41a32060aa0b12e4e491e results in an awkward two line display:
Screenshot_20220729_034037
 
 Can you either move the bracket into \<a> (see Paytaca right above) or into a \<div> (see Paytaca in the mobile section)?

**Review by imaginaryusername — APPROVED** [review]

**Pull request merged**